### PR TITLE
Add Rust Talk 7 to blog

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add blog post
- Embed YouTube video
- Add og-image
- Add header

### CODE REVIEW

File: src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  Benno van den Berg, Principal Software Engineer at Fiberplane, and Luca Palmieri, Principal Engineering Consultant at Mainmatter, discussed Fiberplane’s 3-year Rust journey building an observability product and interactive notebook with Rust.
+  </p>
+
+image: "/assets/images/posts/2024-04-04-fiberplanes-three-year-rust-journey/header.jpg"
+imageAlt: "Benno's picture on a gray background"
+---
+
+Benno kicks off the interview by tracing his personal experience switching from
+.NET to Rust and the beginning of his career at Fiberplane. He then describes
+Fiberplane’s current offer, which includes a collaborative notebook product and
```
Comment (LukeMathWalker) [inline src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md]:
```suggestion
Fiberplane’s current offering, which includes a collaborative notebook product and
```
Maybe?

File: src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+an observability product, Autometrics.
```
Comment (LukeMathWalker) [inline src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md]: Let's add a link to Autometrics here.

File: src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md (hunk 1 above)
Comment (LukeMathWalker) [inline src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md]: Let's link to Fiberplane's website.

File: src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Exploring Fiberplane's 3-Year Rust Journey"
+authorHandle: algo_luca
+tags: [rust]
+bio: "Luca Palmieri, Principal Engineering Consultant"
```
Comment (marcoow) [inline src/posts/2024-04-04-fiberplanes-three-year-rust-journey.md]:
```suggestion
bio: "Principal Engineering Consultant"
```

**Review by marcoow — APPROVED** [review]
all good except one detail

**Pull request merged**